block0.go

```diff
@@ -40,26 +41,24 @@ func readBlock0(filename string) (block0, error) {
 	}
 	defer f.Close()
 
-	b0 := make([]byte, block0Size)
-
-	if _, err := f.Read(b0); err != nil {
-		return nil, err
-	}
-
-	return block0(b0), nil
+	var b0 [block0Size]byte
```

Comment:
Explicitly declaring an array and then taking a slice of it doesn't pass muster. Leaving it as a slice is better.

Reply:
I think that one less allocation, but I obey.